scootapi/client/test_targets_cmd.go

```diff
@@ -0,0 +1,48 @@
+package client
+
+import (
+	"fmt"
+	"github.com/scootdev/scoot/scootapi/gen-go/scoot"
+	"github.com/spf13/cobra"
+	"log"
+	"strconv"
+)
+
+func makeTestTargetsCmd(c *Client) *cobra.Command {
+	r := &cobra.Command{
+		Use:   "test_targets",
+		Short: "Test targets",
+	}
+	r.Flags().StringVar(&c.addr, "addr", "localhost:9090", "address to connect to")
+	r.RunE = func(cmd *cobra.Command, args []string) error {
+		return c.testTargets(cmd, args)
+	}
+	return r
+}
+
+func (c *Client) testTargets(cmd *cobra.Command, args []string) error {
+	client, err := c.Dial()
+	if err != nil {
+		return err
+	}
+	jobDef := scoot.NewJobDefinition()
+	jobDef.Tasks = make(map[string]*scoot.TaskDefinition)
+	// create a task for each target to test
+	for i, t := range getTargets() {
+		task := scoot.NewTaskDefinition()
+		task.Command = scoot.NewCommand()
+		task.Command.Argv = []string{"./pants", "test", t + ":"}
```

Comment:
Long story, but the : is part of the target name; eventually we'll find multiple targets in one package, so just put it in the list above